xml/System.Runtime.InteropServices/ComWrappers.xml

```diff
@@ -198,12 +217,20 @@
         <Parameter Name="wrapper" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="externalComObject">To be added.</param>
-        <param name="flags">To be added.</param>
-        <param name="wrapper">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="externalComObject">Object to import for usage into the .NET runtime.</param>
+        <param name="flags">Flags used to describe the external object.</param>
+        <param name="wrapper">The <see cref="T:System.Object" /> to be used as the wrapper for the external object</param>
+        <summary>Get the currently registered managed object or uses the supplied managed object and registers it.</summary>
+        <returns>Returns a managed object associated with the supplied external COM object.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
```

Comment:
The remarks section should be reverted to "To be added.". I put the exception info in the exception section.